completions/command_opts.sh

```diff
@@ -8,7 +8,7 @@ default_opts() {
     local cmd corecommands=() opts=()
 
     for cmd in "${have_optdump[@]}"; do
-        mapfile -t opts < <(bash "../lib/aur-${cmd}" --dump-options | LC_ALL=C sort)
+        mapfile -t opts < <(bash "$(find ../lib/ -name aur-${cmd})" --dump-options | LC_ALL=C sort)
```

Comment:
Without quoting `$(find ...)` the code will depend on wordsplitting and file glob expansion working as expected.

Instead consider not even using an inner subshell `$()` and using find with `-exec` directly:

```
mapfile -t opts < <(find ../lib -name 'aur-*' -exec bash -- {} --dump-options ';' | LC_ALL=C sort)
```